stock_partner_delivery_window/models/stock_picking.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+from odoo.tools.misc import format_datetime
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+    delivery_date_warning_message = fields.Char(
+        compute="_compute_delivery_date_warning_message"
+    )
```

Comment:
Field name is maybe too common, could conflict with other modules installed in the DB, I propose:
```suggestion
    partner_delivery_window_warning = fields.Char(
        compute="_compute_partner_delivery_window_warning"
    )
```